backend/src/v1/configuration/configuration.controller.ts

```diff
@@ -22,8 +24,32 @@ export class ConfigurationController {
 
   @Get(':configurationCode')
   @Roles(Role.COS_OFFICER)
-  findOne(@Param('configurationCode') configurationCode: string) {
-    return this.configurationService.findOne(configurationCode);
+  async findOne(
+    @Param('configurationCode') configurationCode: string, 
+    @Token() token) 
+  {
+    const result = await this.configurationService.findOne(configurationCode);
+
+    //If configuration is code table version, call another case managment api
+    if(configurationCode === 'CDTABLEVER') {
+      const { data } = await get(token, { 
+        query : '{getConfigurationCode(configuration_code: "CDTABLEVER"){configuration_code configuration_value  active_ind}}'
+      });
+      const caseData = {
```

Comment:
@Scarlett-Truong you might want to check that result is both not null and that there are elements in the results array. There may be a scenario where that can cause an error when creating complaintData.

Reply:
yeah, good point 😃